docs/builds.rst

```diff
@@ -46,7 +46,12 @@ we will first look for a ``mkdocs.yml`` file in the root of your repository.
 If we don't find one,
 we will generate one for you.
 
-Then MkDocs will build any files with a ``.md`` extension.
+Then MkDocs will build any files with a ``.md`` extension within the directory specifed as ``docs_dir`` in the ``mkdocs.yml``. 
+
+If no ``mkdocs.yml`` was found in the root of your repository and we generated one for you, 
+MkDocs will attempt to set ``docs_dir`` by sequentially searching for a  ``docs``, ``doc``, ``Doc``, or ``book`` directory. 
```

Comment:
I think here it should say `Read the Docs will attempt` instead of `MkDocs will attempt` since this is something that RTD code does, right?

Reply:
I agree-- that sounds better. I'll add that change!